src/linux/seccomp.rs

```diff
@@ -14,61 +14,461 @@ const ARCH: TargetArch = TargetArch::x86_64;
 #[cfg(target_arch = "aarch64")]
 const ARCH: TargetArch = TargetArch::aarch64;
 
-/// Seccomp filter.
+/// Seccomp network filter.
 #[derive(Default)]
-pub struct Filter {
-    rules: BTreeMap<i64, Vec<SeccompRule>>,
+pub struct NetworkFilter {
+    allow_networking: bool,
 }
 
-impl Filter {
-    pub fn new() -> Self {
-        Self::default()
+impl NetworkFilter {
+    pub fn new() -> Result<Self> {
+        Ok(Self { allow_networking: false })
     }
```

Comment:
Why is this fallible now? It never actually returns `Err(_)`...

Reply:
Just an oversight due to multiple API changes.